praw/models/reddit/modmail.py

```diff
@@ -41,13 +43,34 @@ def parse(cls, data, reddit):
 
     @classmethod
     def _convert_conversation_objects(cls, data, reddit):
+        """Convert messages and mod actions to PRAW objects."""
         result = {'messages': [], 'modActions': []}
         for object in data['conversation']['objIds']:
             key = object['key']
             object_data = data[key][object['id']]
             result[key].append(reddit._objector.objectify(object_data))
         return result
 
+    @classmethod
+    def _convert_user_summary(cls, data, reddit):
+        """Convert dictionaries of recent user history to PRAW objects."""
+        parsers = {'recentComments':
+                   reddit._objector.parsers[reddit.config.kinds['comment']],
+                   'recentConvos': ModmailConversation,
+                   'recentPosts':
+                   reddit._objector.parsers[reddit.config.kinds['submission']],
+                   }
+        for kind, parser in parsers.items():
+            kind_data = data[kind]
+            for k, v in kind_data.items():
+                v['id'] = k.rsplit('_', 1)[-1]
+                del v['permalink']
+            # Sort by id, oldest to newest
+            sorted_kind = sorted(
+                kind_data.values(),
+                key=lambda x: int(x['id'], base=36), reverse=True)
+            data[kind] = [parser(reddit, _data=x) for x in sorted_kind]
+
```

Comment:
The problem with this approach is that the `Comment` objects have their `_fetched` attribute set to `True`, since there's no `id` parameter. Alternatives:

1. Write separate logic for handling comments.
2. Don't use the summary data provided by the modmail endpoint.
3. Initialize each object using ID, then add the extra information via setattr.